### ISSUE TITLE
Unicode line drawing in PuTTY with Noto Mono

### ISSUE THREAD

**Issue by xnoreq** [googlei18n/noto-fonts#757]
PuTTY with "Use Unicode line drawing code points" and DejaVu Sans Mono, 10pt:
![2016-10-08](https://cloud.githubusercontent.com/assets/4028859/19213213/59fd5e24-8d65-11e6-8520-03d8eea3637e.png)

With the font changed to Noto Mono:
![2016-10-08 1](https://cloud.githubusercontent.com/assets/4028859/19213214/5b471bee-8d65-11e6-94ec-91f3fbf3573a.png)

Here's the portion copied out of the terminal:
```
──────────────┬─────────────
              │1............
    0.3|221  1│
```

Can we get nice unicode line drawing with Noto Mono?

**Comment by xnoreq** [comment]
Box Drawing Unicode block: http://www.unicode.org/charts/PDF/U2500.pdf

**Comment by jungshik** [comment]
@dougfelt

**Comment by dougfelt** [comment]
Ref #617

**Comment by jungshik** [comment]
Is there anything in this bug that won't be resolved when we take care of #617?

**Comment by dougfelt** [comment]
Closed as duplicate of #617.
